applications/FluidDynamicsHydraulicsApplication/python_scripts/apply_hydraulic_inlet_process_test.py

```diff
@@ -0,0 +1,180 @@
+import csv
```

Comment:
You've put the test in the `python_scripts` folder.

Reply:
Oh Sorry, I did a mistake. Now, it is on test folder